pkg/apis/stats/v1alpha1/types.go

```diff
@@ -146,3 +146,49 @@ type RuleTrafficStats struct {
 	Name         string       `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
 	TrafficStats TrafficStats `json:"trafficStats,omitempty" protobuf:"bytes,2,opt,name=trafficStats"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +resourceName=nodelatencystats
+// +genclient:onlyVerbs=create
```

Comment:
This is not right IMO: even though this PR needs only `create`, we will also need `get`, `list`, and maybe `watch` for clients to consume the API. I think `delete` would be good as well: in a _future_ PR, we _may_ want to delete all these resources when the feature is disabled (when a user deletes the `NodeLatencyMonitor` CR). IMO, the only verbs we will not need are `update` & `patch`.

So I'd suggest changing this to `+genclient:onlyVerbs=create,delete,get,list,watch`

Reply:
You will need to regenerate the code after this change

Reply:
Thanks, it is indeed necessary in the scenario you mentioned, I have added and regenerated them.